.github/workflows/cli_core_unit_test.yaml

```diff
@@ -0,0 +1,61 @@
+name: Tanzu CLI Unit Tests
+
+on:
+  pull_request:
+    branches: [main, release-*]
+  push:
+    branches: [main, release-*]
+
+jobs:
+  build:
+    name: Tanzu CLI Unit Tests
+    runs-on: ubuntu-latest
+    steps:
+      - name: Check out code into the Go module directory
+        uses: actions/checkout@v1
+
+      - name: Set up Go 1.x
+        uses: actions/setup-go@v3
+        with:
+          go-version: 1.19
+        id: go
+
+      - name: go cache
+        uses: actions/cache@v3
+        with:
+          path: |
+            ~/.cache/go-build
+            ~/go/pkg/mod
+          key: ${{ runner.os }}-go-${{ hashFiles('**/go.sum') }}
+          restore-keys: |
+            ${{ runner.os }}-go-
+
+      - name: Setup dependencies
+        run: |
+          make tools
+          echo "${PWD}/hack/tools/bin" >> $GITHUB_PATH
+          echo "${PWD}/bin" >> $GITHUB_PATH
+
+      - name: Run gomod
+        run: |
+          make gomod
+
+      - name: Run lint
+        run: |
+          make lint
+
+      - name: Run Unit Test
+        run: |
+          make test > ./make_test.output
```

Comment:
When looking at the new output in GH when running CI, the tests output don't seem to be there anymore because they are redirected to a file.
I worry this will make it hard to know why a test has failed.
Maybe using `tee` instead would be better.  That way it would go both to stdout and the file.

Reply:
thanks for the suggestion, i have updated with `tee` command to stdout the test cases execution output